lib/datapacksjob.js

```diff
@@ -851,6 +865,9 @@ DataPacksJob.prototype.runQueryForManifest = async function(queryInput) {
         }
 
         var query = this.vlocity.omnistudio.updateQuery(queryData.query).replace(/%vlocity_namespace%__/g, this.vlocity.namespacePrefix);
+        query = query.replace("from", "FROM");
+        const subQuery = query.split('FROM');
```

Comment:
This will need just a little extra logic. It should be `replace(" from ", " FROM ")` and split on " FROM " only if the query contains " FROM " since right now it will just break queries which contain "FRom" or "fromDate".

Reply:
@rutlabaga Done. Thanks